backend/main.py

```diff
@@ -224,25 +229,153 @@ def get(self):
         
         return {'files': sorted(list(children))}
 
+class DeployResource(Resource):
```

Comment:
I think you should break this file into separate files. Maybe put all the Storage... classes in one file and put DeployResource in another file.